PgCache_ContentGrabber.php

```diff
@@ -590,7 +591,7 @@ private function _can_read_cache() {
 		 */
 		if ( !$this->_check_ua() ) {
 			$this->cache_reject_reason = 'User agent is rejected';
-			if ( isset( $_REQUEST['w3tc_rewrite_test'] ) ) {
+			if ( isset( Util_Request::get_string( 'w3tc_rewrite_test' ) ) ) {
```

Comment:
Revert to use `isset( $_REQUEST[''w3tc_rewrite_test] )` or use `! empty()`.

Reply:
Changed to ! empty.